Resources/Private/Templates/Search/ExtendedSearch.html

```diff
@@ -20,68 +20,84 @@
 
-    <div class="searchForm">
-        <f:form action="search" name="query" class="form-inline">
-            <div class="form-group form-group-show-deleted">
-                <label for="showDeleted" class="show-label">{f:translate(key:
-                    'search.form.label.inactiveDeleted')}</label>
-                <f:form.checkbox property="showDeleted" class="form-control" id="showDeleted" value="true"
-                                 checked="{query.showDeleted}"/>
-            </div>&nbsp;
-            <div class="main-search-fields">
-            <div class="form-group">
-                <label for="id">{f:translate(key: 'search.form.label.datasetIdentifier')}</label>
-                <f:form.textfield property="_id" class="form-control"
-                                  placeholder="{f:translate(key: 'search.form.label.datasetIdentifier')}" id="id"
-                                  value="{query._id}"/>
-            </div>
-            <div class="form-group">
-                <label for="title">{f:translate(key: 'search.form.label.title')}</label>
-                <f:form.textfield property="title" class="form-control"
-                                  placeholder="{f:translate(key: 'search.form.label.title')}" id="title"
-                                  value="{query.title}"/>
-            </div>
-            <div class="form-group">
-                <label for="author">{f:translate(key: 'search.form.label.author')}</label>
-                <f:form.textfield property="author" class="form-control"
-                                  placeholder="{f:translate(key: 'search.form.label.author')}" id="author"
-                                  value="{query.author}"/>
-            </div>
-            <div class="form-group">
-                <label for="corporation">{f:translate(key: 'search.form.label.corporation')}</label>
-                <f:form.textfield property="corporation" class="form-control"
-                                  placeholder="{f:translate(key: 'search.form.label.corporation')}" id="corporation"
-                                  value="{query.corporation}"/>
-            </div>
+    <div class="alert alert-success fade show alert-save-extended-search-success" role="alert" style="display:none;">
+        {f:translate(key: 'manager.search.saveQuery.successfully')}
+    </div>
+
+    <div class="btn-toolbar mb-3" role="toolbar" aria-label="Toolbar with button groups">
+        <div class="btn-group mr-2" role="group">
+        <div class="dropdown" id="extended-search-add-field">
+            <button class="btn btn-sm btn-secondary dropdown-toggle" type="button"
+                    data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                {f:translate(key: 'manager.search.addSearchField')}
+            </button>
+            <div class="dropdown-menu scrollable-menu" id="search-field-selection" aria-labelledby="dropdownMenuButton">
```

Comment:
Dropdown items not translated?

Reply:
The missing translations have been added.